ci/build/build-vscode.sh

```diff
@@ -13,7 +13,7 @@ main() {
 
   # extensions-ci compiles extensions and includes their media.
   # compile-web compiles web extensions. TODO: Unsure if used.
-  yarn gulp extensions-ci compile-web "vscode-server${MINIFY:+-min}"
+  yarn gulp extensions-ci compile-web "vscode-reh-web${MINIFY:+-min}"
```

Comment:
✍🏼 reh = remote extension host (note for future us)

Reply:
Looks like this does not exist, maybe we want `vscode-web` and `vscode-web-min`?

Reply:
I believe this exists as a generated command. 

https://github.com/microsoft/vscode/blob/8b8e1d1dfaf6fa6e42c4ca2743c40a012c4ac455/build/gulpfile.reh.js#L365

Reply:
Locally I get the same as CI here https://github.com/cdr/code-server/runs/4051125133?check_suite_focus=true#step:8:10

It looks like we are missing the platform part.  If we add it we would not be able to share the build step between platforms though, maybe we can get by with the general build.

Reply:
Ahh this makes sense since upstream uses a separate build pipeline for each platform, and these gulp commands are only defined by the machine running the build. Maybe we could interpolate the platform name into a build step?